app/src/main/java/at/aau/edu/wizards/ui/gameboard/ServerGameBoardViewModel.kt

```diff
@@ -50,15 +52,20 @@ class ServerGameBoardViewModel(
     }
 
     override fun sendMessage(move: String) {
-        viewModelScope.launch {
-            if (gameModel.receiveMessage(move)) {
-                viewModelScope.launch {
-                    server.getConnections().filterIsInstance(ServerConnection.Connected::class.java)
-                        .forEach {
-                            server.send(it, move)
-                        }
+        if (move == END_COMMAND) {
+            _scoreboard.value = true
+        } else {
+            viewModelScope.launch {
+                if (gameModel.receiveMessage(move)) {
+                    viewModelScope.launch {
```

Comment:
No need to start another coroutine with `launch` here. We are already in one. That's currently like starting another thread and the start another one in it. That can have tricky side effects sometimes.

Reply:
thats a misshap from my branch, so my bad here - still can you fix it @martinfri144? - just delete line 58 ( & 69) - thanks

Reply:
done